### PR TITLE
chore: update angular example to rules_js 1.0.0-rc.0

### PR DESCRIPTION
**Pull request by gregmagolan**
@jbedard Something broke with the updated symlink guards. My guess is that it was escaping the sandbox previously.

### CODE REVIEW

File: angular/BUILD.bazel [hunk 1]
```diff
@@ -1,5 +1,3 @@
-load("@aspect_bazel_lib//lib:copy_to_bin.bzl", "copy_to_bin")
-load("@aspect_bazel_lib//lib:jq.bzl", "jq")
```
Comment (jbedard) [inline angular/BUILD.bazel:2]: Opps 👍

**Review by jbedard — APPROVED** [review]

**Comment by alexeagle** [comment]
> Something broke with the updated symlink guards
means this shouldn't merge, right? we should have CI on this repo...

**Comment by gregmagolan** [comment]
Yea, its broken and shouldn't merge

**Comment by jbedard** [comment]
Actually this is still failing, I thought it worked 🤔

**Comment by alexeagle** [comment]
I wonder if it's the same failure as https://github.com/aspect-build/rules_js/pull/265
can you try patching out that commit https://github.com/aspect-build/rules_js/commit/249fcebdb830cc6eb448e37c09b725421cf5df46 ?

**Comment by jbedard** [comment]
That seems likely actually. That issue would cause some files to not make it into the sandbox?

**Comment by alexeagle** [comment]
yes, exactly, and I'd hope that's the only accidental breaking change in the last release

**Review by alexeagle — APPROVED** [review]
Rebased on rc.1 with the revert, bazel build is working for me

**Pull request merged**